cip-0046/cip-0046.md

```diff
@@ -0,0 +1,159 @@
+<pre>
+  CIP: 0046
+  Title: Featured App Reward Markers
+  Author: Moritz Kiefer <moritz.kiefer@digitalasset.com>
+  License: CC-1.0
```

Comment:
@waynecollier-da Do you know if our legal team is OK with DA making CIPs under CC-1.0? I remember them not liking CC licenses but it seems to be the recommended license from CIP 00.